Feat/multi input email invite

wbarbee opened the pull request:
## Summary

New `<kyn-email-invite-input>` component, utilizing tags and texteara (spoofs appearance of one contained component).

## ADO Story

[AB#2295799](https://dev.azure.com/Kyndryl/f7f7ab25-06ec-43dc-902d-dee2e157cebd/_workitems/edit/2295799)

## Figma Link

When available

## Checklist

- [x] Used Conventional Commit messages as outlined in the contributing guide.
  - [ ] Noted breaking changes (if any).
- [ ] Documented/updated all props, events, slots, parts, etc with JSDoc.
  - [ ] Ran the `analyze` command to update Storybook docs.
- [x] Added/updated Stories with controls to cover all variants.
- [ ] Ran `test` locally to address any failures.
- [ ] Added/updated the Figma link for the Story's Design tab.
- [ ] Added any new component exports to the src/index.ts file

## Screenshots
* in progress

brian-patrick-3 reviewed (commented):
Before I dig into the code, I had a thought. What if we want to use this for other things besides email? I think now it would be easy enough to pivot it to be more generic, but if we make it email out of the gate now we might box ourselves in.

wbarbee commented:
> Before I dig into the code, I had a thought. What if we want to use this for other things besides email? I think now it would be easy enough to pivot it to be more generic, but if we make it email out of the gate now we might box ourselves in.
> 
> Edit: After looking at the Figma, it looks like it should allow non-email as well, a more generic "multi tag input" or something.

I've changed the naming  to be more generic (`<kyn-multi-input-field>`) and enhanced the inputs ability to accept things besides email addresses using a new . Added a `DefaultInputType` story as well.

brian-patrick-3 commented on src/components/reusable/multiInputField/multiInputField.ts:
```diff
@@ -0,0 +1,888 @@
+
+  /** Force the component into error state (for stories). */
+  @property({ type: Boolean, reflect: true })
+  invalid = false;
```
Not sure we should keep this if we're only using it for story examples.

brian-patrick-3 commented on src/components/reusable/multiInputField/multiInputField.ts:
```diff
@@ -0,0 +1,888 @@
+  /** Mock database of emails to simulate type-ahead suggestions.
+   * @ignore
+   */
+  private static _mockDb: string[] = [
```
Is this being kept?

brian-patrick-3 commented on src/components/reusable/multiInputField/multiInputField.stories.js:
```diff
@@ -0,0 +1,207 @@
+  <kyn-multi-input-field
+    .value=${args.value}
+    .inputType=${args.inputType}
+    .invalidText=${args.invalidText}
```
Entering invalidText in storybook controls doesn't seem to work in some stories.

brian-patrick-3 commented on src/components/reusable/multiInputField/multiInputField.ts:
```diff
@@ -0,0 +1,888 @@
+    }, 100);
+  }
+
+  private async _fetchSuggestions() {
```
How do suggestions work? I'm not really seeing a way to feed them in.

brian-patrick-3 commented on src/components/reusable/multiInputField/multiInputField.scss:
```diff
@@ -0,0 +1,138 @@
+  display: block;
+}
+
+.container {
```
Should there be some sort of focus state on the container when the input field is active?

brian-patrick-3 commented on src/components/reusable/tag/tag.ts:
```diff
@@ -168,7 +168,15 @@ export class Tag extends LitElement {
+      'sea',
+      'lilac',
+      'ai',
+      'disabled',
```
`disabled` is already a prop here, shouldn't be a color option too.

wbarbee replied:
✅

wbarbee commented:
> Observations:
> 
> 1. Longer text overflows the tag
> 
> <img alt="image" width="708" src="https://private-user-images.githubusercontent.com/60390189/459341816-18616d66-3fb9-4f79-8737-b4e7b712bbf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA5NDM3ODAsIm5iZiI6MTc1MDk0MzQ4MCwicGF0aCI6Ii82MDM5MDE4OS80NTkzNDE4MTYtMTg2MTZkNjYtM2ZiOS00Zjc5LTg3MzctYjRlN2I3MTJiYmYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjI2VDEzMTEyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjZGE0YWU5M2VjM2NlYTNjZTgxYzVhMjQxMGRkMzJiZThkN2IyZmEyNDUwYjFkOTI5YjI3OWZmMmRjODE0N2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xLu_L34NscVsgqCv_e7sbepJi9Hq8ODa_KGvvGvY6Qw">

<img width="599" alt="Screenshot 2025-06-26 at 8 11 02 AM" src="https://github.com/user-attachments/assets/265aa5f9-2d08-4370-8129-1b720a6e460a" />
Fixed, thanks!

ChandanBhat15 reviewed (approved):
Meets AC. Its good to go.
